google-cloud-storage/src/main/scala/org/apache/pekko/stream/connectors/googlecloud/storage/StorageObject.scala

```diff
@@ -456,7 +456,7 @@ object StorageObject {
     bucket,
     generation,
     contentType.asInstanceOf[ContentType],
-    Option(contentType).map(_.asInstanceOf[ContentType]),
+    Option(contentType).asInstanceOf[Option[ContentType]],
```

Comment:
maybe more readable?

```suggestion
    Option(contentType.asInstanceOf[ContentType]),
```